### DISCUSSION TITLE
Background color for folder or directory

### DISCUSSION THREAD

**Discussion by nasrulfuad** [elementary/terminal#637] [Q&A]
Hi, how do you guys custom background color for folder or directory, I can't read the text..

![image](https://user-images.githubusercontent.com/39388505/139617959-fd1fa172-55e1-4820-99b5-d52df252e211.png)

**Comment by jeremypw** [comment]
Are you running Terminal on elementary OS?  Have you installed a non-standard theme or shell?  The Terminal does not normally look like your attached image.

**Reply by nasrulfuad** [reply]
I'm using ZSH, but if I switch to bash, it's still same

![image](https://user-images.githubusercontent.com/39388505/139780973-11e1bd90-3dea-46b6-968f-0bda2f356b1b.png)

I tried to change the default config with
`export LS_COLORS="$LS_COLORS:ow=1;34:tw=1;34:"`
and it change the background color

**Comment by jeremypw** [comment]
Have you tried `export LS_COLORS=""` then `dircolors -b` to reset to default colors?  Have you edited the `.bashrc` file in your home directory?  Do you have a file called `.dir_colors` in your home directory which is overriding the default colors?   The default coloring can be seen by running `dircolors -p`.  The only filetype that gives a yellow background on my system is one is setgid.

**Reply by nasrulfuad** [reply]
No, I never edit my `.bashrc` and I don't have a file `.dir_colors` in my home dir

**Reply by jeremypw** [reply]
I am out of ideas in that case :disappointed: - perhaps someone else can help.

**Comment by M3T4Z0D** [comment]
Hi @nasrulfuad, have you solved this problem? I just installed the system and i'm having the same problem.

**Reply by nasrulfuad** [reply]
Try this 
`export LS_COLORS="$LS_COLORS:ow=1;34:tw=1;34:"`

**Comment by jeremypw** [comment]
We are looking at implementing custom color schemes in Terminal which should help.

**Comment by haagensen** [comment]
Same problem, two years later. I'm trying to change terminal's directory colours but guess what, there's no way to change "directory back colour". Just a window saying I can change "red" to "something else that's clearly not red but will still be called red", etc. To me, this window makes no sense at all.
